.github/workflows/autobuild.yml

```diff
@@ -131,3 +123,34 @@ jobs:
           asset_path:               ${{ matrix.config.asset_path }}
           asset_name:               ${{ matrix.config.asset_name }}
           asset_content_type:       application/octet-stream
+
+  ## Builds a Linux flatpack ## 
+  flatpak-builder:
+    name: "Flatpak Builder"
+    runs-on: ubuntu-latest
+    needs: [create_release]
+    container:
+      image: docker.io/bilelmoussaoui/flatpak-github-actions
+      options: --privileged
+    steps:
+    - uses: actions/checkout@v2
+    - uses: bilelmoussaoui/flatpak-github-actions@v2
+      with:
+        bundle: "io.jamulus.Jamulus.flatpak"
+        manifest-path: "distributions/io.jamulus.Jamulus.json"
+    - uses: actions/download-artifact@v2
+      with:
+        name: io.jamulus.Jamulus
+        path: ~/io.jamulus.Jamulus
+    - name: List
+      run: ls ~/; ls ~/io.*/*
+    - name:                       Upload Flatpack
```

Comment:
@nefarius2001 

I forgot to add the upload of flatpak to my PR...

you should add it to master